app/src/main/res/drawable/bottom_navigation_bar_bg.xml

```diff
@@ -43,13 +43,13 @@
     <item>
         <shape android:shape="rectangle">
             <padding android:top="1dp"/>
-            <solid android:color="#50CCCCCC" />
+            <solid android:color="?attr/toolbarBgBorderColor" />
```

Comment:
these are preliminary changes, https://app.asana.com/0/1157893581871903/1176799629392268 will make sure to update these changes should the experiment move forward